src/site/content/en/learn/css/selectors/index.md

```diff
@@ -0,0 +1,486 @@
+
+A HTML element that has a class of `.my-class` will still be matched to the above CSS rule,
+even if they have several other classes, like this:
+
+```html
+<div class="my-class another-class some-other-class"></div>
+```
+
+This is because CSS looks for a `class` attribute that _contains_ the defined class,
+rather than matching that class exactly.
+
+### ID selector
+
+An HTML element with an `id` attribute should be the only element on a page with that ID value.
+You select elements with an
+[ID selector](https://developer.mozilla.org/en-US/docs/Web/CSS/ID_selectors) like this:
+
+```css
+#rad {
+	border: 1px solid blue;
+}
+```
+
+This CSS would apply a blue border to the HTML element that has an `id` of `rad`, like this:
+
+```html
+<div id="rad"></div>
+```
+
+Similarly to the class selector `.`,
+use a `#` character to instruct CSS to look for an element that matches the `id` that follows it.
+
+{% Aside %}
+If the browser encounters more than one instance of an `id`
+it will still apply any CSS rules that match its selector.
+However, any element that has an `id` attribute is supposed to have a unique value for it,
+so unless you're writing very specific CSS for a single element,
+avoid applying styles with the `id` selector as it means you can't re-use those styles elsewhere.
+{% endAside %}
+
+### Attribute selector
+
+You can look for elements that have a certain HTML attribute,
+or have a certain value for an HTML attribute,
+using the [attribute selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors).
+Instruct CSS to look for attributes by wrapping the selector with square brackets (`[ ]`).
+
+```css
+[data-type="primary"] {
+	color: red;
+}
+```
+
+This CSS looks for all elements that have an attribute of `data-type` with a value of `primary`, like this:
+
+```html
+<div data-type="primary"></div>
+```
+
+Instead of looking for a specific value of `data-type`,
+you can also look for elements with the attribute present, regardless of its value.
+
+```css
+[data-type] {
+	color: red;
+}
+```
+
+```html
+<div data-type="primary"></div>
+<div data-type="secondary"></div>
+```
+
+Both of these `<div>` elements will have red text.
+
+You can use case-sensitive attribute selectors
+by adding an `s` operator to your attribute selector.
+
+```css
+[data-type="primary" s] {
+	color: red;
+}
+```
+
+This means that if a HTML element had a `data-type` of `Primary`,
+instead of `primary`, it would not get red text.
+You can do the opposite—case insensitivity—by using an `i` operator.
+
+Along with case operators,
+you have access to operators that match portions of strings inside attribute values.
+
+```css
+
+/* A href that contains “example.com” */
+[href*="example.com"] {
+	color: red;
+}
+
+/* A href that starts with https */
+[href^="https"] {
+	color: green;
+}
+
+/* A href that ends with .com */
+[href$=".com"] {
+	color: blue;
+}
+```
+
+<figure class="w-figure">
+  {% Codepen {
+    user: 'web-dot-dev',
```

Comment:
Can we use a demo that adds an icon instead of text, since there might be accessibility issue with the text, and the icon is more decorative? Also, maybe make it more obvious with another style shift like a color change for one of them and not the other.

Reply:
I started doing the ol’ inline SVG in CSS for this but backed away because it was potentially opening a can of worms in terms of further explanation. I’ll add some bitmap icons instead though.